code/modules/spells/spell_types/conjure.dm

```diff
@@ -84,4 +84,5 @@
 	return ..()
 
 /obj/effect/proc_holder/spell/targeted/conjure_item/proc/make_item()
-	return new item_type
+	item = new item_type
+	return item
```

Comment:
Would you mind adding a new line at the end of the file to prevent this in the future?